### DISCUSSION TITLE
why disable private network repo?

### DISCUSSION THREAD

**Discussion by ktlcove** [artifacthub/hub#2160] [Q&A] [answered]
https://github.com/artifacthub/hub/blob/118b92528f78c215f549ad309873bea91b9f87f1/internal/util/http.go#L79-L81

we provide a private deploy but disable private network repo .  I want know the reason.

**Comment by tegioz** [comment, marked as answer]
Hi @ktlcove 

If you are running your own Artifact Hub instance, you can control this behavior with the [restrictedHTTPClient](https://artifacthub.io/packages/helm/artifact-hub/artifact-hub?modal=values-schema&path=restrictedHTTPClient) configuration option of the Helm chart.

The reason to provide such a feature is to try to prevent [SSRF](https://owasp.org/Top10/A10_2021-Server-Side_Request_Forgery_%28SSRF%29/) attacks.

**Reply by ktlcove** [reply]
thanks ,  I changed this helm value, It works.
